pages/dao/[symbol]/proposal/components/instructions/Symmetry/SelectBasketModal.tsx

```diff
@@ -0,0 +1,88 @@
+import Modal from "@components/Modal"
+import { PublicKey } from "@solana/web3.js";
+import { BasketsSDK } from "@symmetry-hq/baskets-sdk";
+import { useEffect, useState } from "react"
+
+
+export const SelectBasketModal = ({
+  open,
+  onClose,
+  managedBaskets,
+  sdk,
+  onSelect
+}:{
+  open: boolean,
+  onClose: any,
+  managedBaskets: any,
+  sdk: BasketsSDK | undefined,
+  onSelect: any
+}) => {
+  const [searchValue, setSearchValue] = useState('');
+  useEffect(() => {
+    if(searchValue.length > 40){
+      try {
+        const basketAddress = new PublicKey(searchValue);
+        sdk?.loadFromPubkey(basketAddress).then((basket) => {
+          if(basket){
+            onSelect({
+              basket: basket
+            })
+          }
+        });
+      } catch (e) {
+        console.error(e)
```

Comment:
It would be better to notify the user of the error. 

You can use: `import { notify } from '@utils/notifications';`

```
notify({
    type: 'error',
    message:  e.message,
});
```